schema.txt

```diff
@@ -0,0 +1,55 @@
+
+// Users collection schema
+
+Users collection will have 4 optional fields to distinguish them from regular users.
+	 [optional] superadmin: True/False,
+    [optional] admin: True/False,
+    [optional] editor: True/False,
+    [optional] users_assigned: [xxx.ID, xxx.ID,...]
+
+Here's a sample JSON data for an editor:
+Users {
+   name:"Vegeta",
+   email:"vege@dbz.com",
+   password_hash:"12345",
+   profile_pic:"/users/prince.png",
+   editor:true,
```

Comment:
You need not have "super_admin", "admin", etc as separate keys. 
role: "super admin" / "admin" / "editor"